src/test/java/seedu/address/logic/commands/room/InitRoomCommandTest.java

```diff
@@ -24,13 +24,39 @@ void execute_numberOfRooms_notDefined() {
     }
 
     @Test
-    void execute_numberOfRooms_success() {
+    void execute_numberOfRooms_success1() {
```

Comment:
might want to name this better? maybe `execute_middleRangeNumberOfRooms_success`?